src/components/LoanCards/LoanCardImage.vue

```diff
@@ -63,13 +63,40 @@ export default {
 		openInNewTab: {
 			type: Boolean,
 			default: false
+		},
+		imageEnhancementExperimentVersion: {
+			type: String,
+			default: ''
+		},
+		loanImageHash: {
+			type: String,
+			default: ''
 		}
 	},
+
 	computed: {
+		getImageRetinaUrl() {
+			return this.getImageUrl(this.retinaImageUrl, 'w_970,h_720,c_fill,g_faces');
+		},
+
+		getImageStandardUrl() {
+			return this.getImageUrl(this.standardImageUrl, 'w_480,h_360,c_fill,g_faces');
+		},
+
 		linkTarget() {
 			return this.openInNewTab ? '_blank' : '_self';
 		}
-	}
+	},
+
+	methods: {
+		getImageUrl(imageUrl, transformations) {
+			return (this.imageEnhancementExperimentVersion === 'variant-b' ? this.getCloudinaryUrl(transformations) : imageUrl);  // eslint-disable-line
+		},
+
+		getCloudinaryUrl(transformations) {
+			return `https://res.cloudinary.com/kiva/image/upload/${transformations}/e_viesus_correct/e_sharpen:150/remote/${this.loanImageHash}.jpg`; // eslint-disable-line
```

Comment:
I'm not sure that `/image/upload/` part is needed, see https://github.com/kiva/kiva/blob/development/httpdocs/Utils/ImageUtils.php#L735

Reply:
I've updated the PR. Could you verify the transformation order is correct?